params.go

```diff
@@ -434,6 +412,55 @@ func (p *Params) SetDefaults(appLabel, buildVersion, releaseName, releaseAction
 	}
 }
 
+func initializeSidecarDefaults(sidecar *SidecarParams, params *Params) {
```

Comment:
Would be nice to make this a function on the Params object itself in line with all the other functions:

```
func (p *Params) initializeSidecarDefaults(sidecar *SidecarParams)
```